jenkins/Jenkinsfile

```diff
@@ -230,96 +233,99 @@ EOT
             timestamps()
           }
           steps{
-            script{
-              container('shell'){
-                powershell """
-                  Write-Host "Uninstalling existing erlang installs"
-                  Get-Package -Provider Programs -IncludeWindowsInstaller -Name '*erlang*' |% { & \$_.Meta.Attributes["UninstallString"] /S }
-                  if (-Not (Test-Path "${env.ERLANG_INSTALLER_PATH}")) {
-                    Write-Host "Downloading Erlang..."
-                    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-                    Invoke-RestMethod -uri "${env.ERLANG_DOWNLOAD_URL}" -outfile "${env.ERLANG_INSTALLER_PATH}"
-                  } else {
-                    Write-Host "Found" ${env.ERLANG_INSTALLER_PATH} "in cache. Not downloading Erlang."
-                  }
-                  Write-Host "Installing Erlang..."
-                  Start-Process "${env.ERLANG_INSTALLER_PATH}" -ArgumentList /S -NoNewWindow -Wait
-                """
-                powershell """
-                  if (-Not (Test-Path "${env.RABBITMQ_INSTALLER_PATH}")) {
-                    Write-Host "Downloading Rabbit..."
-                    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-                    Invoke-RestMethod -uri "${env.RABBITMQ_DOWNLOAD_URL}" -outfile "${env.RABBITMQ_INSTALLER_PATH}"
-                  } else {
-                    Write-Host "Found" ${env.RABBITMQ_INSTALLER_PATH} "in cache. Not downloading RabbitMQ."
-                  }
-                  Write-Host "Installing RabbitMQ..."
-                  # Don't wait for rabbitmq installer as it seems to never return when run silently
-                  Start-Process "${env.RABBITMQ_INSTALLER_PATH}" -ArgumentList /S,/QN -NoNewWindow
-                  for (\$attempt=1; \$attempt -le 60; \$attempt=\$attempt+1) {
-                    \$status=(Get-Service RabbitMQ).Status
-                    if ("\$status" -eq "Running") {
-                      Write-Host "RabbitMQ is running."
-                      Break
+            catchError(message: 'Failure on: cloudify-agent tests', buildResult: 'SUCCESS', stageResult: 'FAILURE') {
+              script{
```

Comment:
do we need 'script {' ?

Reply:
I didn't really dig into it, that was working like that before

Reply:
ok, so let's remove it, if no need in script{

Reply:
I'll try to run this pipeline without it and update